frontend/src/components/WofInfoBox.vue

```diff
@@ -0,0 +1,116 @@
+<template>
+    <wof-popper> <!--problem with WofPopper - doesnt work-->
+    <table :class="`WofBox WofBox--${this.type}`">
```

Comment:
Get rid of table structure. Use flexbox to display and position elements properly.
Use css class instead of id. Use BEM naming convention ( http://getbem.com/naming/ )